Update intro_inventory.rst

enwood opened the pull request:
New to ansible, but installing via HomeBrew on Mac OS X (Mountain Lion), I discovered from a note on http://lampros.chaidas.com/index.php/2014/06/22/beginning-with-ansible-on-freebsd-10/ that the inventory file (`hosts`) needs to be in `/usr/local/etc/ansible` not `/etc/ansible`. Submitted change for Intro doc as well. With a hosts file in `/etc/ansible`, ansible did not detect it, and continued to prompt me to specify one with the "-i" parameter. Am I wrong, or is this an OS X anomaly?

enwood committed:
Update intro_inventory.rst

New to ansible, but installing via HomeBrew on Mac OS X (Mountain Lion), I discovered from a note on http://lampros.chaidas.com/index.php/2014/06/22/beginning-with-ansible-on-freebsd-10/ that the inventory file (`hosts`) needs to be in `/usr/local/etc/ansible` not `/etc/ansible`. Submitted change for Intro doc as well. With a hosts file in `/etc/ansible`, ansible did not detect it, and continued to prompt me to specify one with the "-i" parameter. Am I wrong, or is this an OS X anomaly?

srgvg commented:
hi @enwood ; I believe your updates are specific for HomeBrew, whilst the docs are correct for most other platforms, at least Linux. I think your updates should leave current docs as is, and only mentioning a special remark for OSX?

enwood committed:
Update intro_inventory.rst

I've separate the note about Mac OS X/Homebrew as you suggested. I think it's important: it was disheartening to follow Installation instructions as written and have the very first ad-hoc command fail!

mpdehaan commented:
Homebrew has made a change I don't agree with, and we don't control this packaging.

let's get homebrew to fix the problem with homebrew.
